tests/saw/spec/handshake/handshake.saw

```diff
@@ -29,7 +29,8 @@
 // llvm_verify commands in this file use What4-based tactics (e.g., w4_unint_z3)
 // rather than SBV-based tactics (e.g., sbv_unint_z3 or z3).
 include "handshake_io_lowlevel.saw";
-import "rfc_handshake_tls12.cry";
+// TLS1.2 is temporarily removed from SAW testing.
+// import "rfc_handshake_tls12.cry";
```

Comment:
I would disable all these proofs in a separate commit / PR. That will make it easier to revert the commit later.